libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayHistoryMapper.kt

```diff
@@ -76,6 +77,25 @@ class ReplayHistoryMapper @JvmOverloads constructor(
                     eventTimestamp = eventTimestamp
                 )
             }
+            "setRoute" -> {
```

Comment:
Should we extract this (and other event names) magic numbers into constants?

Reply:
prefer the literal string because it's easier to look up. the actual string is defined in native but we don't have a way to link to it